### DISCUSSION TITLE
Broken Links for the Gempy Tutorial Material Download - Cannot Download

### DISCUSSION THREAD

**Discussion by CheongYawPeng** [gempy-project/gempy#964] [General]
I am new to GemPy, and I would like to try using the software.
However, the download links in these sites are not working:
https://docs.gempy.org/tutorials/index.html
https://docs.gempy.org/examples/index.html

Please help if there are other links providing the same material.

**Comment by flohorovicic** [comment]
Dear @CheongYawPeng, 

Thanks for the notification! I checked some of the downloads and those worked for me (for example, the notebooks here: https://docs.gempy.org/examples/geometries/a01_horizontal_stratigraphic.html#sphx-glr-examples-geometries-a01-horizontal-stratigraphic-py and here: https://docs.gempy.org/tutorials/z_other_tutorials/video_tutorial_model_1.html#sphx-glr-tutorials-z-other-tutorials-video-tutorial-model-1-py ).

Can you check if these work for you, as well? And/ or provide more details to the ones that are not working?

**Reply by CheongYawPeng** [reply]
Dear @flohorovicic,

Thanks for your instant reply.

For your first link, this is how it looks like:

Accessing using Microsoft Edge:

![Picture11](https://github.com/user-attachments/assets/e85bbdbf-2c28-48a1-9d51-707e694b2541)

Accessing using Google Chrome:

![Picture22](https://github.com/user-attachments/assets/45effe99-652b-41cd-ade5-a95d07036e75)


Perhaps I don't have the permission to download these files from the link, accessing from my office laptop.

I will try with my private laptop later, and will keep you updated.

**Reply by CheongYawPeng** [reply]
Dear @flohorovicic,

It is indeed the issue with the restriction of using the company laptop. I can download the files from home, using my private computer.

Thank you very much for your instant reply.

Best Regards,
Cheong
